### ISSUE TITLE
[icetray] Get rid of outbox and configure step in python modules (Trac #1867)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1859]
An idea to clean up module declaration to minimum of 3 lines

Migrated from https://code.icecube.wisc.edu/ticket/1867
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:53", 
    "description": "An idea to clean up module declaration to minimum of 3 lines", 
    "reporter": "markw04", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067233566919", 
    "component": "combo core", 
    "summary": "[icetray] Get rid of outbox and configure step in python modules", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2016-10-01T09:17:44", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161017T17:22:18: 
* olivas changed owner from "" to "olivas"
* olivas changed component from "infrastructure" to "icecube offline"
* olivas changed milestone from "" to "October 2016"
* olivas changed status from "new" to "accepted"
* olivas changed type from "enhancement" to "cleanup"
* olivas changed summary from "[Bootcamp] Get rid of outbox and configure step in python modules" to "[icetray] Get rid of outbox and configure step in python modules"

**Comment by icecube-trac** [comment]
Trac update at 20170330T19:02:36: olivas changed milestone from "October 2016" to "May 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170503T20:38:44: 
* olivas changed status from "accepted" to "closed"
* olivas commented:

> In changeset 2868:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2868"
> don't throw if the user doesn't implement Configure and add a default outbox if the user doesn't specify one.  closes #1859.
> ```


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:55:33: nwhitehorn commented:

> In changeset 168804:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168804"
> don't throw if the user doesn't implement Configure and add a default outbox if the user doesn't specify one.  closes #1859.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:53: 
* olivas commented:

> Milestone May 2017 deleted


* olivas changed milestone from "May 2017" to ""
